components/brave_shields/browser/ad_block_engine_service.cc

```diff
@@ -253,49 +266,85 @@ void AdBlockBaseService::OnGetDATFileData(base::OnceClosure callback,
     return;
   }
   GetTaskRunner()->PostTask(
```

Comment:
this shouldn't be using a task runner for anything anymore. `AdBlockEngineService` should only run on one thread